persistence/src/main/java/edu/unc/lib/dl/services/OperationsMessageSender.java

```diff
@@ -314,6 +314,32 @@ public String sendEditTypeOperation(String userid, Collection<PID> pids, Resourc
         return getMessageId(msg);
     }
 
+    /**
+     * Sends Update MODS operation message.
+     *
+     * @param userid id of user who triggered the operation
+     * @param pids objects whose MODS changed
+     * @return id of operation message
+     */
+    public String sendUpdateMODSOperation(String userid, Collection<PID> pids) {
```

Comment:
how about sendUpdateDescriptionOperation?  And updateDescription generally.  Doesn't tie us quite as tightly to the schema, for whenever we stop using mods.